xcb-xprint.el

```diff
@@ -198,11 +200,13 @@
 		 (xcb:-fieldref 'len-data))
 	  :type xcb:-list)
    (data :initarg :data :type xcb:-ignore)
+   (pad~0 :initform 4 :type xcb:-pad-align)
```

Comment:
Your script preserved almost all of the code which is good and expected when we are regenerating the files based on a new xcb protocol version. But I wonder about these paddings introduced here. Why haven't they been present before? Did the layouts change?

Reply:
Thanks, found some of the relevant commits, e.g., https://cgit.freedesktop.org/xcb/proto/commit/src/xprint.xml?id=8a7096ce7375b1840aa350a27a092dd4a22f4c6e.